src/js/game/LevelMVC/FacingDirection.js

```diff
@@ -26,11 +26,11 @@ const FacingDirection = Object.freeze({
     return (facing + 4 + (rotation === 'right' ? 1 : -1)) % 4;
   },
 
-  directionToOffset(direction) {
+  directionToOffset: function (direction) {
     switch (direction) {
       case FacingDirection.North: return [0, -1];
-      case FacingDirection.South: return [0, 1];
       case FacingDirection.East: return [1, 0];
+      case FacingDirection.South: return [0, 1];
```

Comment:
super-minuscule nit: it'd be nice to have the order of the directions match all the other `switch`es in this file